### DISCUSSION TITLE
Can generating flutter project be turned off?

### DISCUSSION THREAD

**Discussion by II11II** [serverpod/serverpod#485] [Q&A] [answered]
There can be some use cases when flutter project is not needed
F.e.
1. There is an existing flutter project 
2. The client app is not written in flutter
3. Usage Serverpod as backend without web/mobile app client, (user would create API and use it for bots)

Maybe to generate `serverpod_flutter` and `serverpod_client` there will be added extra params to the command
F.e. 
 `serverpod create mypod --flutter&&--client`
then `serverpod create mypod` will create only backend related code

**Comment by vlidholt** [comment, marked as answer]
This would be a nice to have. In the mean time, you can just do a `serverpod create` and delete the flutter directory. That should work fine too!

**Comment by yahu1031** [comment]
Doesn't the module do the same? If not, Is that because modules don't have DB table creation?

**Comment by AkshayNG** [comment]
`serverpod create --template module my_module_name`

This command (using --template module) created client and server folders only. Checking further for its use cases.

**Comment by vlidholt** [comment]
A module is just a component of the server. It does not contain the server code itself.

More information here: https://docs.serverpod.dev/concepts/modules
